webhook/psbinding/index_test.go

```diff
@@ -0,0 +1,205 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless requ ired by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package psbinding_test
```

Comment:
Yeah, this was because of a dependency cycle because of how I defined `BindableStatus` (and `knative.dev/pkg/testing/duck` needing to import `psbinding`.  I have split the `Bindable` interface so that the bulk of it lives in `pkg/duck` and `psbinding` implements an extended form with `Do`/`Undo`.